libs/shared/tailwind-ui/src/lib/run-running/run-running.component.tsx

```diff
@@ -0,0 +1,17 @@
+import { Icon } from '../icon';
+
+function RunRunning() {
+	return (
+		<div className="flex gap-4">
+			<Icon name="TriangleExclamationMark" size={48} className="text-primary" />
+			<div>
+				<h1 className="text-2xl font-bold">Run is in progress</h1>
```

Comment:
It's better to say that because of current technical limitations logs will be available only after full run (all tests in the run) will be done.